consutil/main.py

```diff
@@ -31,7 +31,17 @@ def line():
 @click.argument('linenum')
 def clear(linenum):
     """Clear preexisting connection to line"""
-    click.echo("clear line linenum")
```

Comment:
Is there any input parameter linenum validation?
what if we run `clear line swefswef`, what it will return?

Reply:
Yes - `checkDevice(linenum)` (which is in `consutil/lib.py`) checks that there's actually a device for that line number. If there's not, it prints out an error. Right now it also calls `sys.exit()` which might not be ideal - maybe `checkDevice()` should just return an error, which `clear()` can catch and return from instead of exiting?

Reply:
Thank you for the detailed explanations. I'd say it's good enough to exit in checkDevice() function.